app/src/main/java/com/example/playlistmaker/data/db/dao/TrackDao.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.playlistmaker.data.db.dao
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.example.playlistmaker.data.db.entity.TrackEntity
+
+@Dao
+interface TrackDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertTrack(track: TrackEntity)
+
+    @Delete()
```

Comment:
🍏 Если никакие аргументы не передаются, то можно писать аннотацию без ()
```suggestion
    @Delete
```